### ISSUE TITLE
Is it possible to set indicator by text cursor and/or mouse pointer?

### ISSUE THREAD

**Issue by kai-tyro** [pqrs-org/ShowyEdge#22]
Hi, 
Thank you for the wonderful software. I have tried to search the same software on Mac as [Aml Maple](http://amlpages.com/amlmaple.shtml) on windows which use text cursor and/or mouse pointer to be the indicator for the current input source.
Could it be possible to add a cursor presentation option of using the cursor?
Cheers

**Comment by originalrusyn** [comment]
I would like to have such feature too. Some cursor with language code or colorized cursor would be great.

**Comment by wana4** [comment]
I totally agree with your idea.
I hope Apple has some API to be able to implement this feature.

**Comment by samxyzabc** [comment]
A great idea I am waiting for a long time, but doubt Apple has such API, hope anyone can confirm.

于 2021年5月22日 GMT+08:00 上午12:20:41, wanashi ***@***.***> 写到:
>I totally agree with your idea.
>I hope Apple has some API to be able to implement this feature.
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/pqrs-org/ShowyEdge/issues/22#issuecomment-846078121

-- 
使用 K-9 Mail 发送自我的Android设备。

**Comment by weranda** [comment]
> Hi,
> Thank you for the wonderful software. I have tried to search the same software on Mac as [Aml Maple](http://amlpages.com/amlmaple.shtml) on windows which use text cursor and/or mouse pointer to be the indicator for the current input source.
> Could it be possible to add a cursor presentation option of using the cursor?
> Cheers

Just try this app: https://bitbucket.org/avkorotkov/youtype/wiki/Home
It shows the language icon near the cursor.

**Comment by samxyzabc** [comment]
YouType is great, but I don’t know how to make third-party input source work with this it, the link on the FAQ page is dead. Anyone knows how to?

**Comment by weranda** [comment]
> YouType is great, but I don’t know how to make third-party input source work with this it, the link on the FAQ page is dead. Anyone knows how to?

You can ask the developer about it in the issues section.

**Comment by GanZhiXiong** [comment]
It seems that apple doesn't provide an api for text cursor position.

**Comment by yanjieee** [comment]
Is there a way to display it on the window, instead of the title bar? That would also be nice.

**Comment by eugenesvk** [comment]
> Is there a way to display it on the window

good idea. For example, if you could put an indicator in the middle of a vertical left border, this would be closer to your focus and more visible when typing.

Or maybe you could even recolor/overlay the whole window border in 2/3 colors to make it even more visible regardless of where your cursor is
